python/ray/tune/examples/xgboost_example.py

```diff
@@ -34,6 +35,10 @@ def train_breast_cancer(config):
 
 
 if __name__ == "__main__":
+    # Disable strict metric checking to allow final report of accuracy
+    # without reporting the eval logloss
+    os.environ["TUNE_DISABLE_STRICT_METRIC_CHECKING"] = "1"
```

Comment:
The main question I have is if we should support something like `tune.report(metric=..., add=True)` or `tune.report_amend()` or something like that. Basically a method where we duplicate the last passed result and just add the kwargs to it.

We already support this via the `__duplicate__=True` magic parameter, but maybe we would like to make that explicit.

Reply:
I think it is a bit confusing to provide this. `__duplicate__` is actually only used for the edge case at the very end of training to indicate termination.

Reply:
Got it. Then let's leave it as is.